const handleError = (err) => {
    console.error(err)
    process.exit(0)
}

module.exports = handleError